### ISSUE TITLE
Save snapshots to specific directories or drives for data redundancy

### ISSUE THREAD

**Issue by loxoron218** [linuxmint/timeshift#379]
**Is your feature request related to a problem? Please describe.**

I'm quite frustrated looking for a way to regularly send snapshots to specific directories or external drives. Currently, _Timeshift_ only allows snapshots to be saved to a default system drive, which limits my ability to back up my data to external storage for added redundancy and protection against system failures or data loss.

**Describe the solution you'd like**

I think it would be awesome if _Timeshift_ allowed us to specify the directory or external drive where snapshots are saved. This would enable me to regularly back up my system data to an external drive, providing greater flexibility and data redundancy. Ideally, the feature should allow us to select external drives directly from the _GUI_ and terminal to set up automatic backups easier.

**Describe alternatives you've considered**

As an alternative, I've considered manually copying snapshots to external drives after creation, but this is time-consuming and not automated. Additionally, I’ve thought about using _rsync_ and _cronie_ on top of _Timeshift_ , but that defeats my purpose of  having as less tools as possible to back up my data, since I can already use _rsync_ with _cronie_ to send my selected folder to an external drive.

**Additional context**

I suppose this has already been requested before, but I couldn't find it in the existing issues. I hope this feature request is theoretically possible and thanks for your hard work! :)

**Comment by ygerlach** [comment]
Yes, streaming btrfs snapshots to different drives using `btrfs send` would be quite nice. They need to be readonly for that.
